qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -203,13 +203,27 @@ def result(
             results again in another instance or session might fail due to the
             job having been consumed.
 
+            In the case a job fails, but partial results exist, the result
+            method may return a `Result` object. If this is the case, precaution
```

Comment:
Can you clarify that partial results are only returned if `partial=True` is specified? And that one can verify whether some experiments failed by looking at the `result.success` attribute?